src/main/java/org/jitsi/jigasi/TranscriptionGateway.java

```diff
@@ -54,10 +129,54 @@ public TranscriptionGatewaySession createOutgoingCall(CallContext ctx)
                     this,
                     ctx,
                     new GoogleCloudTranscriptionService(),
-                    new LocalTxtTranscriptHandler());
+                    this.handler);
         outgoingSession.addListener(this);
         outgoingSession.createOutgoingCall();
 
         return outgoingSession;
     }
+
+    /**
+     * Get whether to send results in JSON
+     *
+     * @return true if results are send in json, false otherwise
+     */
+    private boolean getSendInJSON()
+    {
+        return JigasiBundleActivator.getConfigurationService()
+            .getBoolean(P_NAME_SEND_RESULT_JSON, SEND_RESULT_JSON);
+    }
+
+    /**
+     * Get whether to send results in TXT
+     *
+     * @return true if results are send in txt, false otherwise
+     */
+    private boolean getSendInTxt()
+    {
+        return JigasiBundleActivator.getConfigurationService()
+            .getBoolean(P_NAME_SEND_RESULT_TXT, SEND_RESULT_TXT);
+    }
+
+    /**
+     * Get whether to save transcript in JSON
+     *
+     * @return true if saved in json, false otherwise
+     */
+    private boolean getStoreInJson()
+    {
+        return JigasiBundleActivator.getConfigurationService()
+            .getBoolean(P_NAME_SAVE_TRANSCRIPT_JSON, SAVE_TRANSCRIPT_JSON);
+    }
+
+    /**
+     * Get whether to save transcripts in txt
+     *
+     * @return true if saved in txt, false otherwise
+     */
+    private boolean getStoreInTxt()
+    {
+        return JigasiBundleActivator.getConfigurationService()
+            .getBoolean(P_NAME_SAVE_TRANSCRIPT_TXT, SAVE_TRANSCRIPT_TXT);
+    }
```

Comment:
I like that the configuration is isolated to one file, but I wonder if TranscriptionGateway is the best place for it. Do you think it would make sense to move it to TranscriptHandler?